### PR TITLE
Add ArrowNorth component

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/ArrowNorth/ArrowNorth.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    return (
+      <div
+        className={className}
+        style={{ transform: `rotate(${rotationOffset}deg)` }}
+      >
+        {circled ? (
+          <ArrowNorthCircle className={className} />
+        ) : (
+          <ArrowNorthSimple className={className} />
+        )}
```
Comment (oterral) [inline src/components/ArrowNorth/ArrowNorth.js]: can you put Arrow component as default ` children` Props.

File: src/components/ArrowNorth/ArrowNorth.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import 'jest-canvas-mock';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ArrowNorth from './ArrowNorth';
+
+configure({ adapter: new Adapter() });
+
+describe('CanvasSaveButton', () => {
```
Comment (oterral) [inline src/components/ArrowNorth/ArrowNorth.test.js]: ArrowNorth

File: src/components/ArrowNorth/ArrowNorth.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * This component displays an arrow pointing the North of the map.
+ */
+class ArrowNorth extends PureComponent {
+  render() {
+    const { className, circled, rotationOffset } = this.props;
+
+    return (
+      <div
+        className={className}
+        style={{ transform: `rotate(${rotationOffset}deg)` }}
```
Comment (oterral) [inline src/components/ArrowNorth/ArrowNorth.js]: Should n't the arrow react when  map's rotation change ?

File: src/components/ArrowNorth/ArrowNorth.js (hunk 1 above)
Comment (edellucien) [reply src/components/ArrowNorth/ArrowNorth.js]: Do we want to allow to change the children?

**Pull request merged**